tests/v2/validation/auth/userretention/userretention_delete_user_test.go

```diff
@@ -0,0 +1,252 @@
+//go:build (validation || infra.any || cluster.any || sanity) && !stress && !extended
+
+package userretention
+
+import (
+	"testing"
+	"time"
+
+	"github.com/rancher/norman/types"
+	"github.com/rancher/rancher/tests/v2/actions/auth"
+	"github.com/rancher/shepherd/clients/rancher"
+	"github.com/rancher/shepherd/extensions/users"
+	"github.com/rancher/shepherd/pkg/session"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	rbac "k8s.io/api/rbac/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type URDeleteTestSuite struct {
+	suite.Suite
+	client  *rancher.Client
+	session *session.Session
+}
+
+func (ur *URDeleteTestSuite) SetupSuite() {
+	logrus.Info("Setting up URDeleteTestSuite")
+	ur.session = session.NewSession()
+
+	logrus.Info("Creating new Rancher client")
+	client, err := rancher.NewClient("", ur.session)
+	require.NoError(ur.T(), err)
+	ur.client = client
+}
+
+func (ur *URDeleteTestSuite) TearDownSuite() {
+	logrus.Info("Tearing down URDeleteTestSuite")
+	ur.session.Cleanup()
+}
+
+func (ur *URDeleteTestSuite) TestDefaultAdminUserIsNotDeleted() {
+	logrus.Info("Starting TestDefaultAdminUserIsNotDeleted")
```

Comment:
we can remove this log and all the logs in subsequent tests as in the jenkins results as they are a bit redundant.